static/app/data/platformPickerCategories.tsx

```diff
@@ -33,6 +33,7 @@ const browser: Set<PlatformKey> = new Set([
   'dart',
   'javascript',
   'javascript-angular',
+  'javascript-astro',
```

Comment:
Adding it here will add it to the project creation flow. Which will break as long as there are no docs. Pls add it in the onboarding docs follow-up PR.
```suggestion
```

Reply:
Thanks, good catch!